README.md

```diff
@@ -10,9 +10,12 @@ Add your name and and one other line here:
 
 Jameel Briones
 - I love cats!
+
 David Kirwan
 Hi I'm currently bashing my head off OSx upgrades for OSx and XCode
 
-Dominik Kawka
-I like lasagna.
+Maciej Marchel
```

Comment:
Did this change get added manually or was it included through a rebase?

Reply:
It was through a rebase, I'll do it again

Reply:
It's fine if it was added through the rebase. could be just github picking up the chnage funny because the change is similar.